cmd/BUILD.bazel

```diff
@@ -1,20 +1,20 @@
-load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
 
 pkg_tar(
     name = "apiserver-builder",
     extension = "tar.gz",
     package_dir = "apiserver-builder",
     srcs = [
-            "//cmd/apiregister-gen",
-            "//cmd/apiserver-boot",
-            "//cmd/vendor/github.com/kubernetes-incubator/reference-docs/gen-apidocs",
-            "//cmd/vendor/k8s.io/kubernetes/cmd/libs/go2idl/client-gen",
-            "//cmd/vendor/k8s.io/kubernetes/cmd/libs/go2idl/conversion-gen",
-            "//cmd/vendor/k8s.io/kubernetes/cmd/libs/go2idl/deepcopy-gen",
-            "//cmd/vendor/k8s.io/kubernetes/cmd/libs/go2idl/defaulter-gen",
-            "//cmd/vendor/k8s.io/kubernetes/cmd/libs/go2idl/informer-gen",
-            "//cmd/vendor/k8s.io/kubernetes/cmd/libs/go2idl/lister-gen",
-            "//cmd/vendor/k8s.io/kubernetes/cmd/libs/go2idl/openapi-gen",
+        "//cmd/apiregister-gen",
+        "//cmd/apiserver-boot",
+        "@com_github_kubernetes_incubator_reference_docs//gen-apidocs",
+        "@io_k8s_code_generator//cmd/client-gen",
+        "@io_k8s_code_generator//cmd/conversion-gen",
+        "@io_k8s_code_generator//cmd/deepcopy-gen",
+        "@io_k8s_code_generator//cmd/defaulter-gen",
+        "@io_k8s_code_generator//cmd/informer-gen",
+        "@io_k8s_code_generator//cmd/lister-gen",
+        "@io_k8s_kube_openapi//cmd/openapi-gen",
```

Comment:
1. the old rule is stale. it misses two additional binaries `go-to-protobuf` and `protoc-gen-gogo` https://github.com/kubernetes-sigs/apiserver-builder-alpha/blob/b5145a29d4adda2da0e45e30f1e0d01111a73e3c/cmd/apiserver-builder-release/main.go#L291-L292

2. i ran the bazel commands successfully locally but the archived path should be `/bin/{apiregister-gen, apiserver-boot, client-gen,...}` instead of flattened on the root `/{apiregister-gen, apiserver-boot, client-gen,...}`.

Reply:
Resolve them as below:
- add bin prefix
- add `go-to-protobuf`, `protoc-gen-gogo`
- add `vendor.tar.gz` packaging
- also temporarily remove `BUILD.bazel` in `pkg/**` since they may confuse the vendor packaging for `pkg/**`. In the future, if I find a better way to use bazel for `pkg/**`, I will add it back.